website/blog/2024-05-22-latest-dbt-stability-improvement-innovation.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Latest dbt: a stable foundation for improvement & innovation"
+description: "Over the past 6 months, we've laid a stable foundation for continuously improving dbt."
+slug: latest-dbt-stability
+
+authors: [michelle_ark, chenyu_li, colin_rogers]
+
+tags: [dbt Cloud]
+hide_table_of_contents: false
+
+date: 2024-05-02
+is_featured: true
+---
+
+As long as it's existed, dbt Cloud has required users to select a version of dbt Core to use under the hood in their jobs and environments. This made sense in the earliest days, when dbt Core minor versions often included breaking changes. It provided a clear way for everyone to know which version of the underlying runtime they were getting.
+
+However, this came at a cost. While bumping a project's dbt version *appeared* as simple as selecting from a dropdown, there was real effort required to test the compatibility of the new version against existing projects, package dependencies, and adapters. On the other hand, putting this off meant foregoing access to new features and bug fixes in dbt.
+
+But no more. Today, we're ready to announce the general availability of a new option in dbt Cloud: **"Keep on latest version."** 
+
+For customers, this means less maintenance overhead, faster access to bug fixes and features, and more time to focus on what matters most: building trusted data products. This will be our stable foundation for improvement and innovation in dbt Cloud. 
+
+But we wanted to go a step beyond just making this option available to you. In this blog post, we aim to shed a little light on the extensive work we've done to ensure that using "Keep on latest version" is a stable, reliable experience for the thousands of customers who rely daily on dbt Cloud. 
+
+## How we safely deploy dbt upgrades to Cloud
+
+We've put in place a rigorous, best-in-class suite of tests and control mechanisms to ensure that all changes to dbt under the hood are fully vetted before they're deployed to customers of dbt Cloud.
+
+This pipeline has in fact been in place since January! It's how we've already been rolling out patch releases for existing dbt Core versions to dbt Cloud customers, and shipping continuous changes to the hundreds of customers who've selected "Keep on latest version" while it's been in Beta and Preview. In that time, this process has enabled us to prevent multiple regressions before they were rolled out to any customers.
+
+We're very confident in the robustness of this process**. We also know that we'll need to continue building trust with time.** We're sharing details about this work in the spirit of transparency and to build that trust.
+
+Any new change to dbt-core and adapters goes through the following steps before it's available to customers in dbt Cloud:
+
+<Lightbox src="/img/blog/2024-05-02-latest-dbt/testing-deploy-pipeline.png" title="Testing and deploy pipeline" />
```

Comment:
```suggestion
<Lightbox src="/img/blog/2024-05-22-latest-dbt/testing-deploy-pipeline.png" title="Testing and deploy pipeline" />
```

Reply:
screenshot wasn't rendering right
<img width="665" alt="Screenshot 2024-05-02 at 14 59 32" src="https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/a6d3c125-b65c-4e20-8df5-24d298f626dd">

Reply:
committing this directly